### ISSUE TITLE
Flutter support

### ISSUE THREAD

**Issue by ChiefWiggum** [Azure/azure-notificationhubs-android#19]
Are there any samples or packages regarding the use of NotificationHub with Dart/Flutter?

**Comment by ChiefWiggum** [comment]
@jwargo Any update on this? It has been more than 6 months and since then flutter has overtaken xamarin at least on google trends:
https://trends.google.de/trends/explore?cat=13&q=flutter,xamarin,react%20native

**Comment by jwargo** [comment]
@ChiefWiggum No, I'm sorry. It's something we're interested in, and we're talking with customers to gauge their interest, but engineering resources allocated yet.
